docs/source/howto/visualising_graphs/graph1.aiida/repo/8f434346648f6b96df89dda901c5176b10a6d83961dd3c1ac88b59b2dc327aa4

```diff
@@ -0,0 +1 @@
+hi
```

Comment:
what's this file doing here? Is this just a placeholder? I take it it is just one of the nodes in the `graph1.aiida` has this file, but it seems kind of pointless for the graph tutorial. Might as well get rid off it.

Reply:
Yeh no worries, just an artifact of when I created this archive many years ago

Reply:
Alright, if you remove that file from the archive I will approve